native~/Shared/src/UnityAssetAccessor.cpp

```diff
@@ -39,13 +42,16 @@ class UnityAssetResponse : public IAssetResponse {
       : _statusCode(uint16_t(request.responseCode())),
         _contentType(),
         _data(std::move(handler.NativeImplementation().getData())) {
-    System::String contentTypeHeader =
-        request.GetResponseHeader(System::String("Content-Type"));
-    if (contentTypeHeader != nullptr) {
-      this->_contentType = contentTypeHeader.ToStlString();
-      this->_headers.emplace("Content-Type", this->_contentType);
+    System::Collections::Generic::Enumerator0 enumerator =
+        request.GetResponseHeaders().GetEnumerator();
+    while (enumerator.MoveNext()) {
+      std::string key = enumerator.Current().Key().ToStlString();
+      std::string value = enumerator.Current().Value().ToStlString();
+      if (value == "Content-Type") {
+        this->_contentType = value;
+      }
+      this->_headers.emplace(key, value);
```

Comment:
I think using std::move with both key and value here will be beneficial (avoid two heap allocations).